README.md

```diff
@@ -23,7 +23,18 @@ To build and compile the project, follow these steps:
 - Run the following command to build the JAR file: `mvn clean package`
 
 ## Usage
-- To start the program with the following command: `java -jar <your-project>.jar`
+Directory should have file: '<your_input>.txt'
```

Comment:
Ok...

Compile and package your program.
Place the JAR file into the separate dir with input file `test.txt`.
Populate `test.txt` with correct input.
Execute your program.
What do you get?

Reply:
output.txt

Reply:
fix

Reply:
You are wrong. 
You will get an excetion during the execution

Reply:
Why?

Reply:
because i need file.txt with name 'input' only